### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by feikedonia**

### CODE REVIEW

File: i18n/nl/cosmic_term.ftl [hunk 1]
```diff
@@ -0,0 +1,100 @@
+cosmic-terminal = COSMIC Terminal
+new-terminal = Nieuw terminalvenster
+
+# Context Pages
+
+## About
+git-description = Git commit {$hash} op {$date}
+
+## Color schemes
+color-schemes = Kleurenpaletten
```
Comment (feikedonia) [inline i18n/nl/cosmic_term.ftl:10]: 'Kleurenpaletten' might not be the (best) translation

File: i18n/nl/cosmic_term.ftl [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+delete = Verwijderen
+import = Importeren
+import-errors = Importfouten
+
+## Profiles
+profiles = Profielen
+name = Naam
+command-line = Opdrachtregel
+tab-title = Tabbladtitel
+tab-title-description = De standaard tabbladtitel overschrijven
```
Comment (feikedonia) [inline i18n/nl/cosmic_term.ftl:22]: 'tabbladtitel' might also be traslated 'tabbladnaam' or 'naam tabblad'

File: i18n/nl/cosmic_term.ftl [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+add-profile = Profiel toevoegen
+new-profile = Nieuw profiel
+make-default = Als standaard instellen
+working-directory = Werkmap
+hold = Behouden
+remain-open = Geopend houden na het beëindigen van het subproces
```
Comment (feikedonia) [inline i18n/nl/cosmic_term.ftl:28]: 'subproces' sound more professional than 'kindproces'

File: i18n/nl/cosmic_term.ftl [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+default-font = Lettertype
+default-font-size = Grootte lettertype
+default-font-stretch = Breedte lettertype
+default-font-weight = Normale letterdikte
+default-dim-font-weight = Subtiele letterdikte
+default-bold-font-weight = Vette letterdikte
+use-bright-bold = Vette tekst feller weergeven
+
+### Splits
+splits = Splitsen
```
Comment (feikedonia) [inline i18n/nl/cosmic_term.ftl:56]: 'Splitsen' is a verb, not a noun

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**